model_access_restriction/models/ir_model_access_restriction.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class IrModelAccessRestriction(models.Model):
+    _name = "ir.model.access.restriction"
+    _description = "Model Access Restrictions. Not having one will disable access."
+
+    _order = "model_id,name,id"
+
+    name = fields.Char(index=True)
+    active = fields.Boolean(
+        default=True,
+    )
+    model_id = fields.Many2one(
```

Comment:
Please, add a restriction so you cannont select this model.

Reply:
Done